content/python/concepts/built-in-functions/terms/str/str.md

```diff
@@ -0,0 +1,39 @@
+---
+Title: 'str()'
+Description: 'Takes in a value that can be converted into a string, and returns a copy of the value in the string datatype.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Strings'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+Takes in a value that can be converted into a string, and returns a copy of the value in the `string` datatype.
+
+## Syntax
+
+```py
+str()
+```
+
+## Example 1
+
+Use `str()` to change the datatype of a `float` value `314.0` into a string:
+
+```codebyte/python
```

Comment:
Let's only have one (1) codebyte snippet per entry (preferably towards the bottom).